src/js/nodeset.js

```diff
@@ -246,6 +246,7 @@ Nodeset.prototype.remove = function () {
                 nodes: null,
                 repeatPath,
                 repeatIndex,
+                removed: true,
```

Comment:
Hopefully this doesn't live that long. The event is marked as for internal use so it's not a big deal to add and remove from.

In the case of a visible form control, you can look at the parent and see if it exists. In the case of a node without a visible form control, without `removed`, you may know you're in some repeat instance when trying to compute cascade stuff but you don't know whether it's been deleted.